greeter-osm.conf

```diff
@@ -12,15 +12,15 @@ statusfile = ./statusgreeter
 subject = Přivítání
 mainmessage = Ahoj <nick>,
 
-	děkujeme za tvůj první příspěvek do OpenStreetMap. Zároveň si dovolujeme srdečně tě přivítat mezi komunitu nadšenců, kteří podobnými úpravami doplňují či zlepšují mapu už několik let. Pokud je toto spíše jednorázová záležitost, nevadí, možná ve svém okolí znáš někoho, kdo rád chodí do lesa, jezdí na kole nebo je zkrátka nadšenec do map - pak budeme rádi, když mu o OpenStreetMap řekneš. Způsobů, jakými je možné přispět, je mnoho - kromě samotného editování to může být například nahrávání GPS záznamů, nahlašování chyb nebo třeba propagace projektu.
+	děkujeme za tvůj první příspěvek do OpenStreetMap. Zároveň si tě dovolujeme srdečně přivítat mezi komunitu nadšenců, kteří podobnými úpravami doplňují či zlepšují mapu už několik let. Pokud je toto spíše jednorázová záležitost, nevadí, možná ve svém okolí znáš někoho, kdo rád chodí do lesa, jezdí na kole nebo je zkrátka nadšenec do map – pak budeme rádi, když mu o OpenStreetMap řekneš. Kromě samotného kreslení mapy lze pomáhat například nahráváním GPS záznamů, nahlašováním chyb, nebo třeba propagací OpenStreetMap.
 
-	Pokud se chceš na něco zeptat nebo se jenom podělit o své zkušenosti či nápad na zlepšení, tak tě rádi potkáme v české konferenci talk-cz@openstreetmap.org (https://lists.openstreetmap.org/listinfo/talk-cz?language=cs). Další informace o české komunitě: https://openstreetmap.cz/komunita 
+	Pokud se chceš na něco zeptat nebo se jenom podělit o své zkušenosti či nápad na zlepšení, rádi tě potkáme v české konferenci talk-cz@openstreetmap.org (https://lists.openstreetmap.org/listinfo/talk-cz?language=cs). Další informace o české komunitě: https://openstreetmap.cz/komunita 
```

Comment:
Ale tohle mě přijde lepší v té původní formě.

Reply:
Přišlo mi to nadbytečné, ale nemám na to silný názor. Použitelné je obojí.